### PR TITLE
#27 Implement different order types

### PR DESCRIPTION
**Pull request by Geukers**
Added Limit and stop orders (they are the same at the moment), and the type of orders gets logged

### CODE REVIEW

File: src/bullets/portfolio/portfolio.py [hunk 1]
```diff
@@ -28,8 +29,15 @@ def __init__(self,
...
         self.transaction_fees = transaction_fees
+        self.pending_buy_limit_orders = []
+        self.pending_sell_limit_orders = []
+        self.pending_buy_stop_orders = []
+        self.pending_sell_stop_orders = []
 
     def market_order(self, symbol: str, nb_shares: float):
+        return self.__order(symbol, nb_shares, "Market Order")
+
+    def __order(self, symbol: str, nb_shares: float, order_type: str):
```
Comment (SamiMammouche) [inline src/bullets/portfolio/portfolio.py]: Sorry to be picky but you should only have a single underscore there

File: src/bullets/portfolio/portfolio.py [hunk 2]
```diff
@@ -28,8 +29,15 @@ def __init__(self,
         self.data_source = data_source
         self.slippage_percent = slippage_percent
         self.transaction_fees = transaction_fees
+        self.pending_buy_limit_orders = []
+        self.pending_sell_limit_orders = []
+        self.pending_buy_stop_orders = []
+        self.pending_sell_stop_orders = []
 
     def market_order(self, symbol: str, nb_shares: float):
+        return self.__order(symbol, nb_shares, "Market Order")
```
Comment (TristanFecteau) [inline src/bullets/portfolio/portfolio.py]: we should differentiate buy and sell market orders depending on nb_shares value (positive/negative) by changing order_type

File: src/bullets/portfolio/portfolio.py (hunk 1 above)
Comment (Geukers) [reply src/bullets/portfolio/portfolio.py]: I removed all the double underscores from the code

File: src/bullets/portfolio/portfolio.py (hunk 2 above)
Comment (Geukers) [reply src/bullets/portfolio/portfolio.py]: I added it. It's a nice touch.

**Review by TristanFecteau — APPROVED** [review]

**Pull request merged**